packages/main/src/themes/TreeListItem.css

```diff
@@ -83,3 +83,41 @@
 :host([active][actionable]) .ui5-li-tree-toggle-icon {
     color: var(--sapList_Active_TextColor);
 }
+
+/* [ui5-li]: infoState */
+:host([info-state="Warning"]) .ui5-li-info {
```

Comment:
ui5-li-info is not appropriate naming as this the tree item - ui5-li-tree-info is more suitable for a class name.